alf/networks/encoding_networks.py

```diff
@@ -406,12 +406,12 @@ def forward(self, inputs, state):
         """
         Args:
             inputs (torch.Tensor):
-            state (list[tuple]): a list of tuples, where each tuple is a pair
+            state (tuple[tuple]): a tuple of tuples, where each tuple is a pair
```

Comment:
I believe that the state should still be a list

Reply:
Now keep it as list.